bundles/AdminBundle/Resources/public/js/pimcore/object/quantityvalue/unitsettings.js

```diff
@@ -229,14 +234,40 @@ pimcore.object.quantityValue.unitsettings = Class.create({
     },
 
     onAdd: function (btn, ev) {
-        var u = {};
-        //id = -1;
-        this.cellEditing.completeEdit();
-        this.grid.store.insert(0, u);
-        this.cellEditing.startEditByPosition({
-            row: 0,
-            column: 0
-        });
+        Ext.MessageBox.prompt(' ', t('enter_the_name_of_the_new_item'),
```

Comment:
title missing

Reply:
pls search for this line of code. we do it exactly that way at dozens of other places. maybe a refactoring / general improvement task ?

Reply:
well anyway this is misleading, as it asks for "name" which is actually an Id.

Reply:
Also, it would be nice to set default auto incremented Id as value so user doesn't need to take care of it everytime.